html_challenge1.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title> 
+            Sundial </title>
```

Comment:
Reminder: Aim to have HTML tags indented properly much like code. This makes it easier to read later on.